playbooks/roles/ses-rook/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: Ensure kubectl is properly configured
+  include_role:
+    name: common-deployer
+    tasks_from: kubectl-setup
+
+- name: Add soc extra repositories
+  become: yes
+  zypper_repository:
+    name: "{{ item['key'] }}"
+    repo: "{{ item['value'] }}"
+    autorefresh: True
+    auto_import_keys: yes
+    state: present
+  loop: "{{ ses_rook_repos | dict2items }}"
+  tags:
+    - ses-rook-repository
```

Comment:
What is this tag good for?

Reply:
for retrying the same step without running the other steps, so you can re-run the same step again and again for whatever reason you wan (only that task its failing, you are tweaking it and want to test that task again, it timed out due to network issues, etc...)

Reply:
Similar to `--start-at-task` when calling ansible-playbook?

Reply:
I guess so (never heard of that) but this still allows you to trigger random tasks, instead of starting from task 4 and containing in a lineal fashion. This allows you to only trigger tasks with a given tag(s)